src/controllers/commands/CommandController.cpp

```diff
@@ -3211,6 +3211,111 @@ void CommandController::initialize(Settings &, Paths &paths)
 
     this->registerCommand("/shield", &commands::shieldModeOn);
     this->registerCommand("/shieldoff", &commands::shieldModeOff);
+
+    this->registerCommand("/shoutout", [](const QStringList &words,
+                                          auto channel) {
```

Comment:
I would prefer if this command was moved to its own file under `src/controllers/commands/builtin/twitch` (see `ShieldMode.hpp`/`ShieldMode.cpp`)